api-client/src/runs/types.ts

```diff
@@ -47,6 +47,7 @@ export interface LegacyGoodRunData {
   modules: LoadedModule[]
   protocolId?: string
   labwareOffsets?: LabwareOffset[]
+  runTimeParameterValues?: RuntimeParameterCreateData
```

Comment:
i know this isn't introduced by this pr but nit:
```suggestion
  runTimeParameterValues?: RunTimeParameterCreateData
```

Reply:
👍  I'll change the casing and audit its uses everywhere